models/ios/logging_global/ios_logging_global.yaml

```
@@ -0,0 +1,54 @@
+module: ios_logging_global
+version_added: 2.0.1
+short_description: Manages logging attributes of Cisco IOS network devices
+description: This module manages the logging attributes of Cisco IOS network devices.
+author: Sagar Paul (@KB-perByte)
+options:
+  config:
+    description: A list containing dictionary of logging options
+    type: list
+    elements: dict
+    suboptions:
+      name:
+        description:
+        - The hostname or IP address of the destination.
+        - Required when dest=host.
+        type: str
+        required: False
+      size:
+        description:
+        - Size of buffer. The acceptable value is in range from 4096 to 4294967295 bytes.
+        type: int
+        required: False
+      dest:
+        description:
+        - Destination of the logs.
+        - On dest has to be quoted as 'on' or else pyyaml will convert to True before it gets to Ansible.
+        type: str
+        required: True
+        choices: ['on', 'host', 'console', 'monitor', 'buffered', 'trap']
+      level:
+        description:
+        - Set logging severity levels.
+        type: str
+        required: False
+        choices: ['emergencies', 'alerts', 'critical', 'errors', 'warnings', 'notifications', 'informational', 'debugging']
+      facility:
+        description:
+        - Set logging facility.
+        type: str
+        required: False
```

```suggestion
        required: False
```
Explicit mention of `required: False` is not needed, so u can remove the respective line from all of available parama